### DISCUSSION TITLE
Unexpected text positioning weirdness

### DISCUSSION THREAD

**Discussion by kd8lvt** [cc-tweaked/CC-Tweaked#618] [Q&A]
When using the Term and Monitor APIs, sometimes text isn't positioned correctly. I specifically get this error when using Lyqyd's Touchpoint API, though looking at his code nothing like this should be happening, and using Touchpoint on 1.7.10 works without issue.

Essentially, an Y value of 2 means "the second block" not "the second character". Here's an example program so you can try it out.
Touchpoint - pastebin get pFHeia96 touchpoint
Example Program - pastebin get hgM0eknG test

What it looks like when happening:
![image](https://user-images.githubusercontent.com/5531395/103038609-dcd17180-453c-11eb-8886-5c3d85ef4b54.png)
(The program should be listing all of the strings in the table, not just the first one)

I'm running MC 1.12.2, CC:T version 1.89.2

Sorry if this is known/not your fault, and thanks for taking the time to look into it!

**Comment by SquidDev** [comment]
Moving this into a discussion, as I'm fairly sure this isn't a bug.

The big thing which stands out is that you're passing a button width/height, while the API actually expects a min/max x/y coordinate. So `t:add(..., 1,k,1,1, ...)` should actually be `t:add(..., 1,k,1,k, ...)`, etc...
